### PR TITLE
add message about ssh key

### PR DESCRIPTION
**Pull request by DylanWatson44**

### CODE REVIEW

File: docs/images/devtools-golang-v1beta1.md [hunk 1]
```diff
@@ -87,16 +90,15 @@ volumes:
 
 ### OCI image building
 
-!!! warning "Security Consideration"
-    OCI building requires that the devtools container run in priviliged mode. For
-    this to be more secure of a rootless OCI runtime (e.g. rootless dockerd) should
-    be used.
+!!! warning "Security Consideration" OCI building requires that the devtools
+    container run in priviliged mode. For this to be more secure of a rootless
+    OCI runtime (e.g. rootless dockerd) should be used.
```
Comment (nhhagen) [inline docs/images/devtools-golang-v1beta1.md:95]: Revert to or it will break formatting. See: https://squidfunk.github.io/mkdocs-material/reference/admonitions/#changing-the-title

```suggestion
!!! warning "Security Consideration"
    OCI building requires that the devtools container run in priviliged mode. For
    this to be more secure of a rootless OCI runtime (e.g. rootless dockerd) should
    be used.
```

File: docs/images/devtools-golang-v1beta1.md [hunk 2]
```diff
@@ -34,17 +37,17 @@ configuration must be compatible.
 
 Interface variables are configurable with environment variables.
 
-If any of the following files are found they will be loaded in the order they are
-listed in before any variables are defined:
+If any of the following files are found they will be loaded in the order they
+are listed in before any variables are defined:
```
Comment (nhhagen) [inline docs/images/devtools-golang-v1beta1.md:41]: Unrelated reformatting

File: docs/images/devtools-golang-v1beta1.md [hunk 3]
```diff
@@ -34,17 +37,17 @@ configuration must be compatible.
...
 - `devtools-settings.mk`
 - `devtools.env`
 - `default.env`,
 - `.env`
 - `${ENVIRONMENT}.env`
 
-If `devtools-targets.mk` is present then it will be loaded after all targets
-are defined.
+If `devtools-targets.mk` is present then it will be loaded after all targets are
+defined.
```
Comment (nhhagen) [inline docs/images/devtools-golang-v1beta1.md:50]: Unrelated reformatting

File: docs/images/devtools-golang-v1beta1.md [hunk 4]
```diff
@@ -87,16 +90,15 @@ volumes:
...
+    container run in priviliged mode. For this to be more secure of a rootless
+    OCI runtime (e.g. rootless dockerd) should be used.
 
 The `Dockerfile` must have a stage named `runtime`, and this is the stage that
 will be pushed when running `publish`.
 
-When pushing docker images credentials from `~/.docker/config.json` will be used,
-and these can be set using commands like:
+When pushing docker images credentials from `~/.docker/config.json` will be
+used, and these can be set using commands like:
```
Comment (nhhagen) [inline docs/images/devtools-golang-v1beta1.md:101]: Unrelated reformatting

File: docs/images/devtools-golang-v1beta1.md [hunk 5]
```diff
@@ -124,14 +126,13 @@ printenv GITHUB_TOKEN \
...
-  OCI image.
+- `BUILD_OCI`: `true` or `false` indicating whether to build an OCI image.
 
   Default: This defaults to `true` if a user supplied `APP_DOCKERFILE` exists
   and to `false` in other cases.
 
-- `OCI_REF_NAMES`: The space seperated OCI reference names that the
-  OCI image should be published as.
+- `OCI_REF_NAMES`: The space seperated OCI reference names that the OCI image
+  should be published as.
```
Comment (nhhagen) [inline docs/images/devtools-golang-v1beta1.md:135]: Unrelated reformatting

**Review by nhhagen — COMMENTED** [review]
This should make it easier for people get started. Could you also create a tech-debt issue that we need to figure out how to support passphrases.

@SaketSaxena if I remember you had a similar issue right?

File: docs/images/devtools-golang-v1beta1.md (hunk 2 above)
Comment (DylanWatson44) [reply docs/images/devtools-golang-v1beta1.md:41]: My auto formatter just fixed the line lengths to be the enforced 80 character limit

**Review by stephendrake — APPROVED** [review]

**Review by nhhagen — APPROVED** [review]
